src/content/self-hosted/administration/configuration.mdx

```diff
@@ -517,6 +517,23 @@ The role Okteto assigns to every user. If empty, Okteto will create a default ro
 clusterRole: "role name"
 ```
 
+### extraRoleBindings
+
+Extra `RoleBindings` to be created for every user's service account. This is useful if you want to grant access to other namespace resources in the cluster.  Disabled by default.
+
+The `RoleBindings` will reference `ClusterRoles` and will be scoped to the namespace specified in the configuration. Here's an example:
+
+```yaml
+extraRoleBindings:
+  enabled: true
+  roleBindings:
```

Comment:
How will this be used? Seems like asking for a existing namespace limits this to very specific scenario. What about giving access to more CRDs, or other types of actions? Is that possible with this?

Reply:
This is to add access to specific namespaces, like `default`. 
Access to CRDs could be cover by:
- Namespaced: https://www.okteto.com/docs/self-hosted/administration/configuration/#clusterrole
- Cluster-wide: https://www.okteto.com/docs/self-hosted/administration/configuration/#globalclusterrole

Reply:
> This is to add access to specific namespaces

Reply:
I'm a bit confused with this change, so let's see if I make sense:

1. With this change, you can now bind a series of roleBindings to every account we create. 
2. We already have a setting to assign a single clusterRoleBinding to every account 
3. Why do we call one 'clusterRole' (where we ask the role's name) and call this 'roleBindings'? -> shouldn't we be calling this extraRoles, since we are the ones that create the roleBinding?
4. Why do we allow only one clusterRole but multiple roleBindings? 
5. Could we simplify the model by having a single setting called 'extraRoles', where the user can add a list of cluster and namespace bindings to be applied to the users we create?
6. Are these applied to the SA the installer uses, or only to the one used during 'okteto up'? (or is it the same?)